<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1351</width>
    <height>659</height>
   </rect>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>100</width>
    <height>129</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>UpArrowCursor</cursorShape>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>DNTC-AD</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/logo.png</normaloff>:/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Open_CAN_Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>91</width>
      <height>381</height>
     </rect>
    </property>
    <property name="text">
     <string>打开CAN</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>371</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>车控参数</string>
    </property>
    <widget class="QPushButton" name="ADAS_Enable_Button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>24</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ADAS</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="Gear_Switch_Button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="currentText">
      <string>档位:无</string>
     </property>
     <item>
      <property name="text">
       <string>档位:无</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>档位:N</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>档位:D</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>档位:R</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>档位:P</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="Acclerate_Reset_Button">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>100</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>归零</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SteeringAngle_Reset_Button">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>140</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>归零</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Fast_Break_Button">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>25</y>
       <width>111</width>
       <height>64</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(204, 0, 0);</string>
     </property>
     <property name="text">
      <string>急刹</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>目标加速度</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>目标转角</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Slow_Break_Button">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>25</y>
       <width>111</width>
       <height>64</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(252, 233, 79);</string>
     </property>
     <property name="text">
      <string>缓刹车</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Acclerate_Send_Value">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>100</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="SteeringAngle_Send_Value">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>140</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QSlider" name="Steer_Slider">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider::groove:horizontal {
        height: 20px;
        background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 rgb(211, 215, 207), stop: 1.0 rgb(211, 215, 207));
}QSlider::handle:horizontal {
        width: 20px;
        background: rgb(0,0,0);
        margin: -6px 0px -6px 0px;
        border-radius: 9px;
}</string>
     </property>
     <property name="minimum">
      <number>-550</number>
     </property>
     <property name="maximum">
      <number>550</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="Acclerate_Slider">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider::groove:horizontal {
        height: 20px;
        background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 rgb(211, 215, 207), stop: 1.0 rgb(211, 215, 207));
}QSlider::handle:horizontal {
        width: 20px;
        background: rgb(0,0,0);
        margin: -6px 0px -6px 0px;
        border-radius: 9px;
}</string>
     </property>
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>651</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="title">
     <string>车辆信息</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QListView" name="Rec_Info_Box">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>611</width>
       <height>151</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Reflash_CPS_COM">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新串口</string>
    </property>
   </widget>
   <widget class="QComboBox" name="GPS_COM_Port">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>请选择</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>串口号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>波特率：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="GPS_BaudRate">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>请选择</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>56000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>460800</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="GPS_Start">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>60</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>连接GPS</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>0</y>
      <width>271</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>GPS配置</string>
    </property>
    <widget class="QPushButton" name="GPS_Data_Save_Button">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>保存GPS数据</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Read_GPS_Data">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>读取GPS数据</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>120</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>算法</string>
    </property>
    <widget class="QPushButton" name="Pure_Pursuit_Start_Button">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>开启Pure_Pursite</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QCustomPlot" name="plot" native="true">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>20</y>
      <width>471</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>490</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>Open_CAN_Button</zorder>
   <zorder>Reflash_CPS_COM</zorder>
   <zorder>GPS_COM_Port</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>GPS_BaudRate</zorder>
   <zorder>GPS_Start</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>plot</zorder>
   <zorder>pushButton</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1351</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/logo.png</normaloff>:/logo.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
